src/core/ddsi/include/dds/ddsi/ddsi_config.h

```diff
@@ -400,7 +400,7 @@ struct ddsi_config
   int retry_on_reject_besteffort;
   int generate_keyhash;
   uint32_t max_sample_size;
-  bool extended_packet_info;
+  int extended_packet_info;
```

Comment:
(A rare case where it would not break things on macOS ppc, since there bool is 4-byte. Usually bools are nightmare, since they are assumed to be 1 byte, but this is not necessarily true.)

Reply:
Cool! I didn't know yet that macOS PPC mapped bool to 32 bits ...